### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by AlyAtalla**
Added hover effects to buttons, Added contact form.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -461,6 +451,13 @@ input::placeholder {
...
+.view-cv:hover {
+  background-color: #008552;
+  color: #fff;
+  cursor: pointer;
+  font-size: 1.2em;
+}
+
 .contact-me {
   display: flex;
   flex-direction: column;
```
Comment (roshan-bajgain) [inline style.css:471]: - [ ] Everything is awesome only a small thing is missing which is a small background image as you need to stick to the design as much as possible. Pleas implement it and then you are good to go.
### Missing part
his

File: index.html [hunk 2]
```diff
@@ -206,10 +206,11 @@ <h1>
     <footer class="footer">
       <ul class="social-icons">
         <li>
-          <a rel="noopener noreferrer" href="https://youtube.com/@alyatalla" target="_blank"><i
-              class="fa-brands fa-youtube"></i></a>
+          <a rel="noopener noreferrer" href="https://www.youtube.com/@alyatalla/" target="_blank"><i
+              class="fa-solid fa-envelope"></i></a>
         </li>
         <li>
+
```
Comment (roshan-bajgain) [inline index.html:214]: - [ ] [Optional] You are designing the mobile version so in the mobile version you will find a horizontal black line at button part. So I would suggest please remove it. It is just like logo of wifi on design.
### Your design

Screenshot 2023-06-06 210553

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @AlyAtalla and @Nelofarzabi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bantealem — APPROVED** [review]
Hi @AlyAtalla  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] No Linters error
- [x] Correct Github flow
- [x] Professional README
## Optional suggestions
- [x] Although you are not suggested by the previous reviewer, I strongly recommend to make sure your design is the same as the Figma design.
Your design

Screenshot from 2023-06-06 19-38-36
Figma design
image
 Please make sure you used the same social media icon and remove the large bottom space of your app.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**